apps/cyberstorm-remix/app/commonComponents/Footer/FooterManagerAd.css

```
@@ -0,0 +1,31 @@
+@layer nimbus-components {
+  .footer-manager {
+    display: flex;
+    flex-direction: column;
+    gap: 4rem;
+    justify-content: end;
+
+    width: 100%;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)